app/src/componentLibrary/sidebar/sidebarButton/sidebarButton.jsx

```diff
@@ -0,0 +1,29 @@
+/*
+ * Copyright 2024 EPAM Systems
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { PropTypes } from 'prop-types';
+
+export const SidebarButton = ({ className, onClick, children }) => (
+  <div className={className} onClick={onClick} role="button" tabIndex={0}>
```

Comment:
Why not to use `button` tag here if we declare it as button?

Reply:
Done.